Resources/Prototypes/tags.yml

```diff
@@ -1309,6 +1309,9 @@
 - type: Tag
   id: WallmountSubstationElectronics
 
+- type: Tag
+  id: WateringCan
```

Comment:
This tag doesn't seem to serve any purpose?

Reply:
It exists to prevent you from crafting the clean bot with it. (And in the future to craft a watering bot.)